conf/conf.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package conf
+
+import (
+	"embed"
+	"io/fs"
+	"strings"
+)
+
+//go:embed **/*
+//go:embed app.ini
```

Comment:
```suggestion
//go:embed **/*
```

I don't think the second line is needed, shouldn't `**/*` include everything?